### PR TITLE
Remove broadway

### PR DESCRIPTION
**Pull request by cv65kr**
This PR contains:
- [x] Removed broadway, rabbitmq lib from composer
- [x] New implementation CQRS + ES based on Symfony Messenger (inspirited by broadway). I am thinking about implementation this in this template, but IMO is too much for that so I created new package [cv65kr/messenger](https://github.com/cv65kr/messenger). Currently working on that (still need some tests, readme, replay etc.), I will be thankful for some CR or contributions, but all tests passed with this template so logic is fine and QA passed.

I decide to give You permission to this repository (obviously if You want), this package was create especially for this template, but everybody can implement this in their app. Sometimes I may slow respond, so You have possiblity accept or modify some part of code.

TODO:
- PHP docker image needed `php7-amqp` https://github.com/jorge07/alpine-php/pull/66

------

Next step bump to Symfony 5.

### CODE REVIEW

File: src/Infrastructure/User/Query/Projections/UserView.php [hunk 1]
```diff
@@ -8,18 +8,18 @@
...
 
-class UserView implements SerializableReadModel
+class UserView implements ReadModelInterface
 {
     /**
      * @throws \App\Domain\Shared\Exception\DateTimeException
      * @throws \Assert\AssertionFailedException
      */
-    public static function fromSerializable(Serializable $event): self
+    public static function fromSerializable(EventInterface $event): self
```
Comment (jorge07) [inline src/Infrastructure/User/Query/Projections/UserView.php]: EventInterface doesn't match the needs here. Serializable it's the projection itself not an event

**Comment by jorge07** [comment]
Thanks for the work here @cv65kr.

I've mixed feelings with this PR. I like the idea of remove Broadway and the implementation of the lib as a symfony bundle. But, I think this bundle needs to get mature in order to migrate. Atm it's a symfony bundle and I would like to have an stand alone line and another as symfony bridge/bundle. Also, there're a couple of thing in the line I would like to discuss.

I think this PR will be living for long until we get the other lib ready. I also have my own implementation in a private repo but never finished. Join forces and get something more mature and up to date its something I really can bet for. I would also know if @Lutacon would like to join in this travel.

Another option it's to open another branch in this repo for the migration and start integrating changes.

**Comment by cv65kr** [comment]
@jorge07 I truly understand Your POV, obviously we can discuss about solution and focus our power to create good library.

So maybe some Slack?

**Comment by jorge07** [comment]
Sure, lets catch up next week

**Pull request closed without merging**